lib/cmds/extension_cmds/delete.js

```diff
@@ -29,27 +28,26 @@ export const builder = (yargs) => {
 }
 
 export async function deleteExtension (argv) {
-  await assertLoggedIn(argv)
-  await assertSpaceIdProvided(argv)
+  const {context, id} = argv
+  await assertLoggedIn(context)
+  await assertSpaceIdProvided(context)
 
-  const { cmaToken, activeSpaceId, activeEnvironmentId } = await getContext()
-  const spaceId = argv.spaceId || activeSpaceId
-  const environmentId = argv.environmentId || activeEnvironmentId || 'master'
+  const { managementToken, activeSpaceId, activeEnvironmentId = 'master' } = context
```

Comment:
I think we can move setting of the activeEnvironmentId to `master` to the middlware

Reply:
✅done 🙂